migrator/models/cdn.py

```diff
@@ -0,0 +1,187 @@
+import datetime
+from enum import Enum
+from typing import List
+
+import sqlalchemy as sa
+from sqlalchemy.ext import declarative
+from sqlalchemy.dialects import postgresql
+from sqlalchemy import orm
+from sqlalchemy_utils.types.encrypted.encrypted_type import (
+    AesGcmEngine,
+    StringEncryptedType,
+)
+
+from migrator.extensions import config
+from migrator.models.common import (
+    Action,
+    RouteType,
+    RouteModel,
+    CertificateModel,
+    OperationModel,
+    AcmeUserV2Model,
+    ChallengeModel,
+    OperationState,
+)
+
+convention = {
+    "ix": "idx_%(column_0_label)s",
+    "uq": "uq_%(table_name)s_%(column_0_name)s",
+    "ck": "ck_%(table_name)s_%(constraint_name)s",
+    "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+    "pk": "pk_%(table_name)s",
+}
+
+metadata = sa.MetaData(naming_convention=convention)
+
+CdnModel = declarative.declarative_base(metadata=metadata)
+
+
+def db_encryption_key():
+    return config.CDN_DATABASE_ENCRYPTION_KEY
+
+
+class CdnUserData(CdnModel):
+    """
+    CdnUserData is about the Let's Encrypt user associated with a CdnRoute.
+    We probably have no reason to ever think about this model in this project.
+    """
```

Comment:
😆 

"Here's this model - pay no attention to it!"

In all seriousness, this is just for helping process and keep track of the data from the Let's Encrypt side, correct?  Just used internally here?